TheFourFifths/consus-client
# [TFF-178] Render model due dates

### Summary

Due dates have already been added to models. This PR renders those due dates in the view.

### Checklist


### How to Review

1. Run this branch with the server on `dev`
2. `rfid:123456`
3. Check out `m8y7nFLsT`
4. Look at the due date

### Links

- [TFF-178](https://msoese.atlassian.net/browse/TFF-178)
